index.html

```diff
@@ -79,7 +79,7 @@ <h2>Multi-Post Stories Gain+Glory</h2>
             <li>JavScript</li>
             <li>html</li>
           </ul>
-          <a href="#button">See Project</a>
+          <a class="project-button" id="2" href="#button">See Project</a>
```

Comment:
- [ ] I recommend aligning buttons to the end of the row as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="622" alt="Screenshot 2022-03-03 at 11 59 46" src="https://user-images.githubusercontent.com/77038610/156521757-de9d2441-afb0-4ebe-a7e9-381447a0930d.png">  | <img width="614" alt="Screenshot 2022-03-03 at 12 00 02" src="https://user-images.githubusercontent.com/77038610/156521794-a532317e-78e7-4ee3-86f0-eca5ebc36677.png">